SCClassLibrary/Common/Files/SoundFile.sc

```diff
@@ -70,6 +71,38 @@ SoundFile {
 		^res
 	}
 
+    *write {|pathName, array, headerFormat, sampleRate, sampleFormat|
```

Comment:
Just one last thing: one can imagine that one may want to call `write` on an instance of SoundFile, to write back its data to a new path, or after some changes. Then this new `*write` method on the class would be a little skewed, because it writes an array.

Maybe call the method `*writeArray`? 

Even better it would be if one could make a new instance of SoundFile from the array and then write that SoundFile instance. But I am not sure right now if this is too complicated to implement.

Reply:
All done.

Thanks for you help ironing this out, there were far more issues than I expected, but I think the code look *much* better for it! Thanks again, J.